modules/machine-config-daemon-metrics.adoc

```diff
@@ -0,0 +1,77 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes/nodes-nodes-machine-config-daemon-metrics.adoc
+
+[id="machine-config-daemon-metrics_{context}"]
+= Machine Config Daemon metrics
+
+Beginning {product-title} 4.3, the Machine Config Daemon provides a set of metrics. These metrics can be accessed using the Prometheus Cluster Monitoring stack.
+
+The following table describes this set of metrics. Note that:
+
+* Metrics marked with `*` in the `Name` and `Description` columns represent serious errors that might cause performance problems. Such problems might prevent updates and upgrades from proceeding.
+* While some entries contain commands for getting specific logs, the most comprehensive set of logs is available using the `oc adm must-gather` command.
+
+[cols="1,1,2,2", options="header"]
+.MCO metrics
+|===
+|Name
+|Format
+|Description
+|Notes
+
+|mcd_host_os_and_version
+|[]string{"os", "version"}
+|Shows the OS that MCD is running on, such as RHCOS or RHEL. In case of RHCOS, the version is provided.
+|
+
+|ssh_accessed
+|counter
+|Shows the number of successful SSH authentications into the node.
+|The non-zero value shows that someone might have made manual changes to the node. Such changes might cause irreconcilable errors due to the differences between the state on the disk and the state defined in the machine configuration.
+
+|mcd_drain*
+|{"drain_time", "err"}
+|Logs errors received during failed drain. *
+|While drains might need multiple tries to succeed, terminal failed drains prevent updates from proceeding. The `drain_time` metric, which shows how much time the drain took, might help with troubleshooting. For further investigation, see the logs by running:
+
+`$ oc logs -f -n openshift-machine-config-operator machine-config-daemon-<hash> -c machine-config-daemon`
+
+|mcd_pivot_err*
+|[]string{"pivot_target", "err"}
+|Logs errors encountered during pivot. *
+|Pivot errors might prevent OS upgrades from proceeding. For further investigation, see the logs by running these commands:
+
+`$ oc logs -f -n openshift-machine-config-operator machine-config-daemon-<hash> -c machine-config-daemon`
+
+`$ journalctl -u pivot.service`
```

Comment:
It should be noted that this command should be run on the nodes themselves after using something like `oc debug node <node>`